prow/cmd/horologium/main.go

```diff
@@ -163,6 +163,13 @@ func sync(prowJobClient prowJobClient, cfg *config.Config, cr cronClient, now ti
 				if _, err := prowJobClient.Create(&prowJob); err != nil {
 					errs = append(errs, err)
 				}
+			} else {
+				fields := make(logrus.Fields)
```

Comment:
nit: Can you have this formatted more like WithFields() on line 142?

Reply:
Just fixed. PTAL. Thanks!

Reply:
Thanks! ^^